src/cpp/session/modules/SessionGit.cpp

```diff
@@ -414,8 +414,18 @@ class Git : public boost::noncopyable
          file.status = line.substr(0, 2);
 
          std::string filePath = line.substr(3);
+
+         // git status --porcelain will quote paths that contain special
+         // characters (e.g. spaces on Windows). strip those characters
+         // when encountered
+         std::string::size_type n = filePath.size();
+         if (n >= 2 && filePath[0] == '"' && filePath[n - 1] == '"')
```

Comment:
Does `git status --porcelain` do any escaping inside the path -- for instance escaping `"` characters in the path? (wondering if we need to do any unescaping here too)